.travis.yml

```diff
@@ -1,7 +1,21 @@
+sudo: false
 language: python
+
 python:
     - "3.3"
     - "2.7"
+
+env:
+  global:
+  - API_CHECK=https://appengine.google.com/api/updatecheck
+
+before_script:
+# Idea from https://blog.bekt.net/p/gae-sdk/
+- SDK_VERSION=$(curl -s $API_CHECK | awk -F '\"' '/release/ {print $2}')
+- SDK_URL=https://storage.googleapis.com/appengine-sdks/featured/google_appengine_$SDK_VERSION.zip
+- wget $SDK_URL -nv
+- unzip -q google_appengine_$SDK_VERSION.zip
```

Comment:
I think the ``PYTHONPATH`` environment variable will need to be updated to see the library. Something like:

```
PYTHONPATH=$PYTHONPATH:`pwd`/google_appengine_$SDK_VERSION/
```

https://docs.travis-ci.com/user/environment-variables/